src/cloud-api-adaptor/ibmcloud-powervs/README.md

```diff
@@ -30,27 +30,33 @@ pvsadm image qcow2ova --prep-template-default > image-prep.template
 
 Add the following snippet to `image-prep.template`
 ```
-yum install -y gcc gcc-c++ git make
+yum install -y gcc gcc-c++ git make wget perl
+wget https://github.com/mikefarah/yq/releases/download/v4.42.1/yq_linux_ppc64le
+chmod +x yq_linux_ppc64le && mv yq_linux_ppc64le /usr/local/bin/yq
 git clone -b main https://github.com/kata-containers/kata-containers.git
 git clone https://github.com/confidential-containers/cloud-api-adaptor.git
-cd cloud-api-adaptor/ibmcloud-powervs/image
+cd src/cloud-api-adaptor/ibmcloud-powervs/image
```

Comment:
shouldn't it be `cd cloud-api-adaptor/src/cloud-api-adaptor/ibmcloud-powervs/image`?

Reply:
Updating